exercise2/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace exercise2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+			try
+			{
+				Console.Write("Ange ett decimaltal:");
+				string input = Console.ReadLine();
+				double value = Convert.ToDouble(input);
+				int closestInt = (int)Math.Round(value);

```

Comment:
```suggestion
				double closestInt = Math.Round(value);
```
Du behöver inte göra denna till en int!